src/components/TodoList/TodoList.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Todo } from '../Todo/Todo';
+import { todoType } from '../propTypes/todoType';
+
+import './TodoList.scss';
+
+export const TodoList = ({ todos }) => (
+  <div className="container">
+
+    <ul className="list-group todos mx-auto text-light">
+      {todos.map(todo => (
+        <li key={todo.id}>
```

Comment:
get rid of second li tag

![image](https://user-images.githubusercontent.com/57598825/97975640-1d7cfc00-1dd2-11eb-8da5-dd81d90213e5.png)